### DISCUSSION TITLE
How to read HL7 message from FTP

### DISCUSSION THREAD

**Discussion by shantanukulkarni-sdn** [nextgenhealthcare/connect#5269] [Q&A]
I've HL7 SIU12 messages for each patient on FTP, I connect to FTP and I want to read messages from FTP, How can I read content from txt file from FTP?

**Comment by jonbartels** [comment]
Show what you have already tried.

Have you looked at the FTP settings in the File Reader connector for MC?

Have you read the MC manual or getting started guides?
